### PR TITLE
Create locales for Hausa and Somali

### LINKED ISSUE #2171: Create moment locale for - Hausa, Somali
**Is your feature request related to a problem? Please describe.**
Moment doesn't have a locale for Hausa and Somali.

**Describe the solution you'd like**
Create a locale for Hausa (ha) and Somali (so) in `psammead-locales`. 
Get the locales [here](https://repl.it/repls/ComplicatedCluelessTab)

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by DenisHdz**
Resolves #2171

**Overall change:**
 Add moment locales for Hausa (ha) and Somali (so).

**Code changes:**
- Add locale files with months and relative time
- Add unit tests.

---

- [X] I have assigned myself to this PR and the corresponding issues
- [X] Automated jest tests added (for new features) or updated (for existing features)
- [X] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.11.0 | [PR#2193](https://github.com/bbc/psammead/pull/2193) Create `ha` (Hausa) and `so` (Somali) locales |
```
Comment (ghost) [inline packages/utilities/psammead-locales/CHANGELOG.md]:
```suggestion
| 2.11.0 | [PR#2196](https://github.com/bbc/psammead/pull/2196) Create `ha` (Hausa) and `so` (Somali) locales |
```

File: packages/utilities/psammead-locales/moment/so.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+var moment = require('moment');
+
+var months = [
```
Comment (ghost) [inline packages/utilities/psammead-locales/moment/so.js]: Can we keep this consistent. By having `months.split('_')`

Comment (DenisHdz) [reply packages/utilities/psammead-locales/moment/so.js]: Happy to use `months.split('_')` for consistency, but I think is more readable in this way.

Comment (ghost) [reply packages/utilities/psammead-locales/moment/so.js]: I agree, but we should ensure consistency across all `moment` overrides

Comment (DenisHdz) [reply packages/utilities/psammead-locales/moment/so.js]: Done :)

Comment (DenisHdz) [reply packages/utilities/psammead-locales/moment/so.js]: Done :)

File: packages/utilities/psammead-locales/README.md [hunk 3]
```diff
@@ -8,27 +8,29 @@ This package provides a collection of locale configs, used in BBC World Service
...
+- `/moment/so` - locale for Somali
 
 The following locales have overrides to meet BBC World Service requirements where these differ from the upstream locale. Note that importing them will also cause the upstream locale to be loaded.
 
 - `/moment/ar` - locale override for Arabic
 - `/moment/fa` - locale override for Persian (Farsi)
+- `/moment/gu` - locale override for Gujarati
+- `/moment/hi` - locale override for Hindi
+- `/moment/ky` - locale override for Kyrgyz
+- `/moment/mr` - locale override for Marath
```
Comment (AlistairGempf) [inline packages/utilities/psammead-locales/README.md]:
```suggestion
- `/moment/mr` - locale override for Marathi
```

**Review by AlistairGempf — APPROVED** [review]
This all looks good other than the typo.

**Review by ghost — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by staylos92** [comment]
LGTM 👍

**Pull request merged**